addons/bestja_page_fixtures_ucw/templates/pages/become_volunteer.xml

```diff
@@ -117,19 +117,19 @@
                                 <div class="col-md-8 col-md-offset-2">
                                     <ul class="like_write">
                                         <li class="mb64">
-                                            <span>
+                                            <a href="http://www.facebook.com/plugins/share_button.php?href=https%3A%2F%2Fwww.facebook.com%2FUCWUW&amp;layout=link"><span>
```

Comment:
This URL is clearly meant to be used in an iframe, and not as a stand alone link (this `<a>` doesn't have a label or a closing tag BTW). If you want to *link* to Facebook - link to the actual Facebook page, as Krzysztof suggested.